### ISSUE TITLE
Implement scrolling announcements for iOS

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#22747]
Once https://github.com/flutter/flutter/pull/21764 is submitted we will have all of the machinary in place for accessibility scrolling announcements. Unlike Android, the API for iOS just takes a string. This will require us to generate the announcement in Flutter for localizations purposes and pass it to iOS via one of our accessibility methods.

Scrolling happens in the widgets layer which doesn't know about text, so we will need to inject a platform specific formatter to produce the message, where the implementation can be provided by CupertinoLocalizations.

**Comment by pedromassangocode** [comment]
Looks like https://github.com/flutter/flutter/pull/21764 is merged now!

**Comment by jonahwilliams** [comment]
This was completed and should have been closed!
